sql/world/updates/20181208-00_spell_coefficient_override.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE IF NOT EXISTS `spell_coefficient_override` (
   PRIMARY KEY (`spell_id`,`min_build`),
   UNIQUE KEY `unique_index` (`spell_id`,`min_build`) USING BTREE,
   KEY `spell_id` (`spell_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8 COMMENT='Spell System';
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci; COMMENT='Spell System';
```

Comment:
We can not change old update files.

Reply:
`Sql query failed due to [You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ', assign_on_target_flag, assign_self_cast_only, assign_c_is_flag, proc_flags, pr' at line 1], Query: [SELECT spell_id, rank, assign_on_target_flag, assign_self_cast_only, assign_c_is_flag, proc_flags, proc_target_selfs, proc_chance, proc_charges, proc_interval, proc_effect_trigger_spell_0, proc_effect_trigger_spell_1, proc_effect_trigger_spell_2 FROM spell_custom_override]`

some update does not load

Reply:
removed mention from rt